packages/superset-ui-demo/storybook/stories/plugins/plugin-filter-antd/Select/Stories.tsx

```diff
@@ -1,42 +0,0 @@
-import React from 'react';
-import { action } from '@storybook/addon-actions';
-import { boolean, withKnobs } from '@storybook/addon-knobs';
-import { SuperChart, getChartTransformPropsRegistry } from '@superset-ui/core';
-import { AntdSelectFilterPlugin } from '@superset-ui/plugin-filter-antd';
-import transformProps from '@superset-ui/plugin-filter-antd/lib/Select/transformProps';
-import data from './data';
-import { withResizableChartDemo } from '../../../../shared/components/ResizableChartDemo';
-import 'antd/dist/antd.css';
```

Comment:
Just a reminder: we need migrate these to the `apache/superset` storybook.

Reply:
ok